modin/pandas/test/dataframe/test_udf.py

```diff
@@ -305,6 +310,24 @@ def test_query(data, funcs):
         df_equals(modin_result, pandas_result)
 
 
+TEST_VAR = 2
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("local_var", [2])
+def test_query_with_local_and_global_var(data, local_var):
```

Comment:
this looks exactly the same as `test_eval_with_local_and_global_var` except the method called on the dataframe... could we collapse those in a single test and just parametrize by the method name? or at least put those next to each other.